### ISSUE TITLE
Scroll within the page

### ISSUE THREAD

**Issue by cordinGH** [YU000jp/logseq-plugin-hierarchy-tool#4]
Thank you very much for developing this plugin. I think the TOC implementation of your plugin is the most comfortable solution, but one drawback is that it doesn't seem to support scrolling within the page when clicking on a heading. However, your other plugin "Left-Sidebar Enhance" does support this feature.

**Comment by YU000jp** [comment]
Your opinion is correct. Originally, the option should have given users a choice. Currently, I am unable to apply my own time resources to this project, so the improvements will not proceed.
This plugin integrates with subpage links and is a simplified TOC feature.

**Comment by cordinGH** [comment]
That's all right, then I'll use your "Left-Sidebar Enhance" plugin.This plugin is also very useful.
